src/components/ui/toast.js

```diff
@@ -1,35 +1,7 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './toast.css';
+import { createStandaloneToast } from "@chakra-ui/react"
 
-export default function toast(title, msg) {
-
-    // create DOM element
-    const toast = React.createElement('div', {
-        className: 'card Toast fadeInUp',
-        style: {
-            position: 'fixed',
-            bottom: '10px',
-            right: '10px',
-            maxWidth: '300px',
-        }
-    }, <>
-        {title && <strong className='text-red-500'>{title}</strong>}
-        <div>{msg}</div>
-    </>)
-
-    // render toast into DOM
-    setTimeout(() => {
-        ReactDOM.render(toast, document.getElementById('toast'))
-    }, 500)
-
-    // remove toast from DOM
-    setTimeout(() => {
-        const toastEl = document.getElementById('toast');
-        ReactDOM.unmountComponentAtNode(toastEl)
-    }, 5000)
-}
+export const toast = createStandaloneToast()
 
 export function actionCompletedToast(msg) {
-    toast('Action completed', msg);
+    toast({ title:'Action Completed', description: msg})
```

Comment:
This is personal preference I guess but I prefer this to be in lowercase. `Action completed`. Mind if I change it?

Reply:
no problem! go on!